docs/src/main/asciidoc/native-reference.adoc

```diff
@@ -252,6 +260,15 @@ For example, we can see how the heap and text sections take most of binary:
 readelf -SW ./target/debugging-native-1.0.0-SNAPSHOT-runner
 ----
 
+[NOTE]
+====
+Runtime containers produced by Quarkus to run native executables will not include the tools mentioned above.
+To explore a native executable within a runtime container,
+it's best to run the container itself and then `docker cp` the executable locally.
```

Comment:
It might be good to include an example, e.g.

```suggestion
it's best to run the container itself and then `docker cp` the executable locally, e.g.:

----
docker run -i --rm --name=mytest -p 8080:8080 test/debugging-native:1.0.0-SNAPSHOT
docker cp mytest:/work/application path/on/host/
----
```

I am not sure you can nest a code block in a note though!

Reply:
Just tried it, yes you can.